UPDATING.md

```diff
@@ -41,6 +41,10 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Change python3 as Dataflow Hooks/Operators default interpreter
+
+Now the `py_interpreter` argument for DataFlow Hooks/Operators has been changed from python2 to python3.
```

Comment:
Can you describe the consequences for the end-user i.e. all jobs is started using Python 3 by default now?

Reply:
I think it's good enough for now @mik-laj -> I think what we really need in UPDATING.md is to record all the changes, and before we release 2.0.0 we should prepare some more complete migration guide for the users.

This should include a tool that will mostly automate migrating DAGs and correcting the DAGs they have where needed and print warnings for all cases that need some manual review and corrections (with appropriate instructions). But I think those instructions should be part of that tool rather than UPDATING.md.

Reply:
I will create an issue about it.

Reply:
https://issues.apache.org/jira/browse/AIRFLOW-6390 created and proposed that in devlist.